### PR TITLE
[akali] Update to v1.41

### PR DESCRIPTION
**Pull request by winsoft666**

### CODE REVIEW

File: ports/akali/portfile.cmake [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
 
+string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "static" AKALI_STATIC)
+
 vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}
 	PREFER_NINJA
     OPTIONS
+        -DAKALI_STATIC:BOOL=${AKALI_STATIC}
         -DBUILD_TESTS:BOOL=OFF
 )
```
Comment (NancyLi1013) [inline ports/akali/portfile.cmake]: Could you place this at the top of this file?
`vcpkg_fail_port_install(ON_ARCH "arm" "arm64" ON_TARGET "UWP")`

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

File: ports/akali/portfile.cmake (hunk 1 above)
Comment (winsoft666) [reply ports/akali/portfile.cmake]: OK, I have updated.

**Review by NancyLi1013 — APPROVED** [review]

**Comment by NancyLi1013** [comment]
Hi @winsoft666 thanks for this PR.
I noticed that this port failed on OSX pipeline from CI system. The failures like this:
```
/Users/vagrant/azure-agent/_work/3/s/buildtrees/akali/src/b27e9b5e42-a471ebc6a4/src/byteorder.cpp:24:77: error: use of undeclared identifier 'htobe16'
void SetBE16(void *memory, uint16_t v) { *static_cast<uint16_t *>(memory) = htobe16(v); }
                                                                            ^
/Users/vagrant/azure-agent/_work/3/s/buildtrees/akali/src/b27e9b5e42-a471ebc6a4/src/byteorder.cpp:26:77: error: use of undeclared identifier 'htobe32'
void SetBE32(void *memory, uint32_t v) { *static_cast<uint32_t *>(memory) = htobe32(v); }
                                                                            ^
/Users/vagrant/azure-agent/_work/3/s/buildtrees/akali/src/b27e9b5e42-a471ebc6a4/src/byteorder.cpp:37:47: error: use of undeclared identifier 'be16toh'
uint16_t GetBE16(const void *memory) { return be16toh(*static_cast<const uint16_t *>(memory)); }
                                              ^
/Users/vagrant/azure-agent/_work/3/s/buildtrees/akali/src/b27e9b5e42-a471ebc6a4/src/byteorder.cpp:39:47: error: use of undeclared identifier 'be32toh'
uint32_t GetBE32(const void *memory) { return be32toh(*static_cast<const uint32_t *>(memory)); }
                                              ^
/Users/vagrant/azure-agent/_work/3/s/buildtrees/akali/src/b27e9b5e42-a471ebc6a4/src/byteorder.cpp:53:47: error: use of undeclared identifier 'htobe16'
uint16_t HostToNetwork16(uint16_t n) { return htobe16(n); }
                                              ^
/Users/vagrant/azure-agent/_work/3/s/buildtrees/akali/src/b27e9b5e42-a471ebc6a4/src/byteorder.cpp:55:47: error: use of undeclared identifier 'htobe32'
uint32_t HostToNetwork32(uint32_t n) { return htobe32(n); }
                                              ^
/Users/vagrant/azure-agent/_work/3/s/buildtrees/akali/src/b27e9b5e42-a471ebc6a4/src/byteorder.cpp:57:47: error: use of undeclared identifier 'be16toh'
uint16_t NetworkToHost16(uint16_t n) { return be16toh(n); }
                                              ^
/Users/vagrant/azure-agent/_work/3/s/buildtrees/akali/src/b27e9b5e42-a471ebc6a4/src/byteorder.cpp:59:47: error: use of undeclared identifier 'be32toh'
uint32_t NetworkToHost32(uint32_t n) { return be32toh(n); }
```
Could you please take a look and try to fix it?

**Comment by ubsan** [comment]
@winsoft666 why did you close?

**Comment by winsoft666** [comment]
I see macos check failed, so I closed it, and commit again.


在 2020年1月11日，08:37，nicole mazzuca <notifications@github.com> ：

﻿

@winsoft666<https://github.com/winsoft666> why did you close?

**Comment by NancyLi1013** [comment]
Hi @winsoft666 thanks for this PR.
The failures on osx seems to be not related this PR. 
So you don't need to close this PR. We are trying to fix the regressions on osx pipeline.

**Comment by NancyLi1013** [comment]
I noticed you have opened another PR #9617 to update this port.
Please add the comments to describe the reason for closing an PR next time.

Thanks.

**Pull request closed without merging**